### ISSUE TITLE
Accessibility Guide Tools

### ISSUE THREAD

**Issue by jfc3-dol** [18F/accessibility#140]
While looking over the Accessibility Guide's Tools page ( https://pages.18f.gov/accessibility/tools/ ), I noticed that you have "DHS's trusted tester" spelled using lowercase and Trusted Tester should be capitalized from everything I have ever seen from DHS ( https://www.dhs.gov/trusted-tester ).

This text can be found when talking about the WAT toolbar.

**Comment by nickbristow** [comment]
https://github.com/18F/accessibility/commit/21bbb56cf7589eb9519131a32564151798ff3d8c
